libs/net/models/Areas/Services/Models/Content/ContentModel.cs

```diff
@@ -282,6 +282,16 @@ public ContentModel(Entities.Content entity) : base(entity)
             entity.Series = new Entities.Series(model.OtherSeries, model.SourceId);
         }
 
+        if (model.Product != null)
+        {
+            entity.Product = new Product(model.Product.Name);
+        }
+
+        if (model.Source != null)
```

Comment:
Why have you added this?  It can result in the creation of new products when saving content.  Which is not desirable behaviour.

Reply:
Removed, thanks